### ISSUE TITLE
Recent annotations list links to audio overly precisely

### ISSUE THREAD

**Issue by atruskie** [Allcharles/workbench-client#1789]
**Describe the bug**

The recent annotations list generates URLs to the listen page that are too short. For example:

<img width="282" alt="image" src="https://user-images.githubusercontent.com/1452968/153305141-36bd9d76-8e58-44c6-bc2d-7313f1154aca.png">

An example URL generated from this list of links:

```
https://www.ecosounds.org/listen/262864?start=1041.6642&end=1042.9645
```

**To Reproduce**

1. Go to any point page
2. Look at the URLs generated in the recent annotations list

**Expected behavior**

As a rule we don't generate links to the listen page (or any media request for that matter) that have offsets which are not whole second.
The idea here is to increase the chance of a cache hit.

Any other media segment is far more likely to hit the cache if the parameters are used `1041 & 1042` over `1041.6642 & 1042.9645`.

Further, the listen page should generally be showing _lots_ of audio, filling the space of the page. If we're sending a user to a spot, we should only be changing the start offset (and let the width end offset auto-scale).

If we really do want to show a single annotation in isolation, then we can send someone to the library page for the annotation.

Make the following changes:

- audit every link to listen to ensure only whole integer offsets are used
   - for start floor to nearest second
   - for end ceiling
- for these recent annotation links:
   - specify only the start parameter
   - in lieu of assuming the width of content on the listen page, and a small buffer to the start offset of the annotation (say 1 to 5 seconds).

**Screenshots**

An inappropriately short listen page:

<img width="559" alt="image" src="https://user-images.githubusercontent.com/1452968/153306159-c26f7c5f-318e-4478-860e-62bf2af016b5.png">

**Comment by Allcharles** [comment]
I thought i just fixed this, i might have my solution in a different branch. I'll check

**Comment by Allcharles** [comment]
~Fixed in #1788~

https://github.com/QutEcoacoustics/workbench-client/blob/143c44206953433e3e49bcfb87500fa801c0b91e/src/app/models/AudioEvent.ts#L95-L104
